onnx/common/status.h

```diff
@@ -4,6 +4,7 @@
 #pragma once
 
 #include <memory>
+#include <ostream>
```

Comment:
it's better to add this within a ifdef? (a macro of new msvc version?)

Reply:
I think not. This header *always* needs the contents of `<ostream>`, adding the `#include` here when it's unnecessary due to transitive inclusion (e.g. on older MSVC versions) has nearly no cost due to include guards and `#pragma once`.